src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let price = days * 40;
```

Comment:
Prefer using 'const' for the 'price' variable since it is not being reassigned, but rather its value is being updated.